src/main/java/com/google/research/bleth/servlets/EnqueueSimulationServlet.java

```diff
@@ -0,0 +1,60 @@
+package com.google.research.bleth.servlets;
+
+import com.google.cloud.tasks.v2.AppEngineHttpRequest;
+import com.google.cloud.tasks.v2.CloudTasksClient;
+import com.google.cloud.tasks.v2.HttpMethod;
+import com.google.cloud.tasks.v2.QueueName;
+import com.google.cloud.tasks.v2.Task;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * A servlet used for enqueuing a task targeted at endpoint '/new-simulation',
+ * in order to create and run a new simulation.
+ */
+@WebServlet("/enqueue-simulation")
+public class EnqueueSimulationServlet extends HttpServlet {
+
+    static final String projectId = "bleth-2020";
+    static final String locationId = "europe-west1";
+    static final String queueId = "simulations-queue";
+
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        try (CloudTasksClient client = CloudTasksClient.create()) {
+            // Construct the fully qualified queue name.
+            String queueName = QueueName.of(projectId, locationId, queueId).toString();
```

Comment:
Can this be changed into a class-wide constant? (It seems to only depend on other constants?)